compiler/src/iree/compiler/Dialect/Flow/Transforms/FuseDequantizationMatmul.cpp

```diff
@@ -10,12 +10,17 @@
 #include "mlir/Dialect/Arith/IR/Arith.h"
```

Comment:
can we move this out to the global optimization pipeline? (not sure if it exactly fits there today, but it's a lot of code to carry in flow for a special case) - thoughts @hanhanW / @MaheshRavishankar ?

Reply:
I think this isnt rebased on main cause I see this pass is already out of Flow.

Reply:
The pass itself lives in Flow/transforms, but it is used in global optimization. I did not move passes to the GlobalOptimization folder because they have been using flow ops. I can move them out to global optimization in my next cleanup round if it is not a concern.